### PR TITLE
Real time output

### PR DESCRIPTION
**Pull request by b-chmiel**
Add new format oireal
STATUS CODE TIME REAL_TIME NVM MEM SYSC

### CODE REVIEW

File: src/seccomp/policy/DefaultPolicy.cc [hunk 1]
```diff
@@ -44,7 +44,9 @@ void DefaultPolicy::addExecutionControlRules(bool allowFork) {
              "clock_nanosleep",
              "open",
              "epoll_create1",
-             "openat"});
+             "openat",
+             "newfstatat",
+             "pread64"});
```
Comment (Wolf480pl) [inline src/seccomp/policy/DefaultPolicy.cc]: `pread64` should go in `addInputOutputRules()`
`newfstatat` should be in `addFileSystemAccessRules()`

File: src/limits/TimeLimitListener.cc [hunk 2]
```diff
@@ -88,24 +88,27 @@ executor::ExecuteAction TimeLimitListener::onSigalrmSignal() {
...
+    auto time = getTimeUsage();
+    return verifyTimeUsage(move(time));
 }
 
 void TimeLimitListener::onPostExecute() {
     // TODO: run this just after child exit
-    verifyTimeUsage();
-    // TODO Save time usage to OutputBuilder.
+    auto time = getTimeUsage();
+    outputBuilder_->setRealTimeMicroseconds(time->realTimeUs);
```
Comment (Wolf480pl) [inline src/limits/TimeLimitListener.cc:98]: I think we should double-check this, but IIRC the old "real-cpu" supervisor uses user time, not real time.

File: src/printer/RealTimeOIOutputBuilder.cc [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    KillReason reason = killReason_;
+    if (reason == KillReason::NONE) {
+        if (killSignal_ > 0 || exitStatus_ > 0) {
+            reason = KillReason::RE;
+        }
+    }
+
+    std::stringstream ss;
+    ss << killReasonName(reason) << " " << exitStatus_ << " "
+       << milliSecondsElapsed_ << " " << realMilliSecondsElapsed_ << " "
```
Comment (Wolf480pl) [inline src/printer/RealTimeOIOutputBuilder.cc]: I see you've added a new field here, but maybe putting realMilliSecondsElapsed in place of either miliSecondsElapsed or that 0ULL below, it'd be easier to adapt sioworkers and oiejq to parse this?

**Review by Wolf480pl — CHANGES_REQUESTED** [review]

File: src/printer/RealTimeOIOutputBuilder.cc (hunk 3 above)
Comment (b-chmiel) [reply src/printer/RealTimeOIOutputBuilder.cc]: You are right, I will replace milliSecondsElapsed

**Review by Wolf480pl — APPROVED** [review]

**Pull request merged**